src/components/Card.tsx

```diff
@@ -12,7 +22,7 @@ export function useCardsData() {
           throw new Error("네트워크 응답이 올바르지 않습니다.");
         }
         const data = await response.json();
-        const processedData = data.folder.links.map((link) => ({
+        const processedData = data.folder.links.map((link: any) => ({
```

Comment:
아까 API의 응답데이터의 타입을 정리해둔것 기억나시나요 ? 
```typescript
export type Link = {
    createdAt: string;
    description: string;
    id: number;
    imageSource: string;
    title: string;
    url: string;
};
```

위의 데이터를 여기 `any`말고 `link`로 쓸 수 있습니다.  이렇게 하나 둘씩 `any`타입을 줄여 나갈 수 있어요.

```suggestion
const processedData = data.folder.links.map((link: Link) => ({
```

Reply:
```typescript
export type FolderResponse = {
    folder: Folder;
};
```

아까 정리해둔 타입을 다시 사용해서 타입을 좀 더 정확하게 쓸 수 있습니다. 

```typescript
const data = (await response.json()) as FolderResponse;
                const processedData = data.folder.links.map((link: Link) => ({
                    ...link,
                    timePassed: timeCalculate(link.createdAt) as string,
                    getDate: getDateText(new Date(link.createdAt)),
                }));
                setCardsData(processedData);

```
<img width="630" alt="스크린샷 2024-03-27 오후 11 27 03" src="https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/assets/40737888/761e46ed-594c-43f1-bc5f-f2f9ad9c214f">

processedData가 `any`에서 정확한 타입형태로 바뀌었어요. ㅎㅎ